calipso/constants.py

```diff
@@ -47,12 +68,15 @@ class Plot(object):
         'polar stratospheric cloud']
 
 LOG_FILENAME = 'log/CALIPSO_debug.log'
-HELP_PAGE = 'http://syntaf.github.io/vocal/'
+HELP_PAGE = 'http://nasa-develop.github.io/VOCAL/developer_index.html'
 
 TIME_VARIANCE = 0.001
 ALTITUDE_VARIANCE = 0.3
 PATH = '.'
-#PATH = os.path.dirname(os.path.realpath(__file__))
+HOMEPATH = expanduser('~')
```

Comment:
You'll need to modify the paths here to follow the Win branch. User application data is stored in the appdata directory, not the installed directory (otherwise, the user would need permission to run the application as it would attempt to write data to the C:/ drive).

The Win path is set to `os.getenv('APPDATA') + '/vocal/fakedir'`